frontend/src/components/maps/leaflet/LayerPopup/LayerPopupContent.tsx

```diff
@@ -75,11 +75,15 @@ export const LayerPopupContent: React.FC<IPopupContentProps> = ({ data, config,
 
   useEffect(() => {
     getLTSAData();
-  }, [data]);
+  }, [data.PID]);
 
   const getLTSAData = async () => {
```

Comment:
maybe we can break this function out into a "reusable" component.....instead of having it in 2 files.